docs/tutorials/semantic_search/asymmetric_embedding_model.md

```diff
@@ -0,0 +1,539 @@
+# Tutorial: Running Asymmetric Semnantic Search within OpenSearch
+
+This tutorial demonstrates how to generate text embeddings using an asymmetric embedding model in OpenSearch which will be used
+to run semantic search. This is implemented within a Docker container, the example model used in this tutorial is the multilingual
+`intfloat/multilingual-e5-small` model from Hugging Face. 
+You will learn how to prepare the model, register it in OpenSearch, and run inference to generate embeddings.
+
+> **Note**: Make sure to replace all placeholders (e.g., `your_`) with your specific values.
+
+---
+
+## Prerequisites
+
+- Docker Desktop installed and running on your local machine.
+- Basic familiarity with Docker and OpenSearch.
+- Access to the Hugging Face model `intfloat/multilingual-e5-small` (or another model of your choice).
+---
+
+## Step 1: Spin Up a Docker OpenSearch Cluster
+
+To run OpenSearch in a local development environment, you can use Docker and a pre-configured `docker-compose` file.
+
+### a. Update Cluster Settings
+
+Before proceeding, ensure your cluster is configured to allow registering models. You can do this by updating the cluster settings via the following request:
+
+```
+PUT _cluster/settings
+{
+  "persistent": {
+    "plugins.ml_commons.allow_registering_model_via_url": "true",
+    "plugins.ml_commons.only_run_on_ml_node": "false",
+    "plugins.ml_commons.model_access_control_enabled": "true",
+    "plugins.ml_commons.native_memory_threshold": "99"
+  }
+}
+```
+
+This configuration ensures that OpenSearch can accept machine learning models from external URLs and can run models across non-ML nodes.
+
+### b. Use a Docker Compose File
+
+You can use this sample [file](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/#sample-docker-compose-file-for-development) as an example.
```

Comment:
Is there any specific directory this file should be in?

Reply:
So as long as the user can find it and run the terminal command its fine. Although they can be organized and create a folder and place it there but I think thats TMI haha